tests/accuracy/test_global_correction.py

```diff
@@ -174,8 +174,8 @@ def test_lhc_global_correct_dpp(tmp_path: Path, model_inj_beams: DotDict, dpp: f
     for key in response_dict.keys():
         assert DELTAP_NAME in response_dict[key].columns
 
-    # create and load fake measurement
-    dpp_path = run_dpp(tmp_path, dpp, model_inj_beams.beam)
+    # create fake measurement from previously created model
+    dpp_path = f"tests/inputs/correction/deltap/twiss_dpp_{dpp:.1e}_B{model_inj_beams.beam}.dat"
```

Comment:
Chiming in before the weekend to ask for this hard-coded path to disappear, and using `CORRECTION_INPUTS` defined at the top of file.

This way it doesn't crash when a test isn't started from the repo's root and if we change some location it'll propagate easier.

Reply:
Oh well, @JoschD committed that change as I was complaining :P

Reply:
Yeah sorry, I committed without thinking!